activation/e2e/poet_test.go

```diff
@@ -229,68 +229,3 @@ func TestSubmitTooLate(t *testing.T) {
 	)
 	r.ErrorIs(err, activation.ErrInvalidRequest)
 }
-
-func TestCertifierInfo(t *testing.T) {
```

Comment:
This test shouldn't be removed but changed to call `client.Info` and verify if the returned data matches expectations.

Reply:
I have modified [this](https://github.com/spacemeshos/go-spacemesh/pull/6274/files#diff-798e24aa415f524699f5e62d8127c70a92eaac5664efe14950b64ab22a915599R261) test, to check what happens if certificate exists or not + can add check if returned values match expectation. 

I do not think, that it makes much sense to test `client.Info` separately, because it returns error only in case if can't parse certificate url, but certificate data doesn't exist at all, it wouldn't return any error.

Reply:
The tests here interact with a real poet server. They exist to ensure our client can talk with an actual poet server.

Reply:
Reverted 1 test (it doesn't make sense to verify with_cert/without_cert, because `client.Info` will not return error in case if  there is no certificate)